app/controllers/companies_controller.rb

```diff
@@ -14,30 +14,35 @@ def create_confirmation
     redirect_to new_user_registration_path
   end
 
-  def update_company
+  def edit_company
+    @current_user = User.find_by_id(current_user)
     @edit_company = Company.find_by_username(session[:company_username])
     @image = @edit_company.image_check
     render "edit"
   end
 
-    def update_company_confirmation
+    def edit_company_confirmation
```

Comment:
Assignment Branch Condition size for edit_company_confirmation is too high. [29.07/15]